paths/release_triggers/create.yaml

```diff
@@ -0,0 +1,91 @@
+---
+summary: Create a release trigger
+description: Create a new recurring release. New releases will be published automatically, based on the cron schedule provided.
+  Currently, only one release trigger can exist per distribution.
+operationId: release_triggers/create
+tags:
+  - Release Triggers
+parameters:
+  - "$ref": "../../parameters.yaml#/X-PhraseApp-OTP"
+  - "$ref": "../../parameters.yaml#/account_id"
+  - "$ref": "../../parameters.yaml#/distribution_id"
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
+        type: object
+        title: release/create/parameters
+        properties:
+          cron_schedule:
+            description: Cron schedule for the scheduler. Read more about the format of this field at https://en.wikipedia.org/wiki/Cron
+            type: string
+            example: "15 18 1,3 * *"
+          time_zone:
+            description: Time zone for the scheduler
+            type: string
+            example: "Europe/Berlin"
+          locale_ids:
+            description: List of locale ids that will be included in the release.
+            type: array
+            items:
+              type: string
+            example:
+              - abcd1234cdef1234abcd1234cdef1234
+              - fff565db236400772368235db2c6117e
+          tags:
+            description: Only include tagged keys in the release. For a key to be included it must be tagged with all tags provided
+            type: array
+            items:
+              type: string
+            example:
+              - android
+              - feature1
+          branch:
+            description: Branch used for release
+            type: string
+            example: my-feature-branch
+          app_min_version:
+            description: The created releases will be available only for apps with version greater or equal to this value
+            type: string
+            example: "1.0.0"
+          app_max_version:
+            description: The created releases will be available only for apps with version less or equal to this value
+            type: string
+            example: "2.0.0"
```

Comment:
I apologise for going back and forth on this PR, I think `platform` should also be documented among other params?

Reply:
hm I don't see where do we assign it upon creation in the `NestedAttributesBuilder`?

Reply:
Ah, we just use the distribution's platforms to fetch the formats. Normal release form has a checkboxes to check which one, this release will be for.